README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
+
+Install Go Task
+
+```sh
+brew install go-task
```

Comment:
Explain what dependencies are being installed by 'task install', as users may not know what to expect.